nestjs/src/auth/auth-user.model.ts

```diff
@@ -38,14 +39,17 @@ export class AuthUser {
 
     user.students?.forEach(student => {
       roles[student.courseId] = 'student';
-      const info = courses[student.courseId] ?? ({ mentorId: null, studentId: null, roles: [] } as CourseInfo);
+      const info =
+        courses[student.courseId] ?? ({ mentorId: null, studentId: null, roles: [], isExpelled: null } as CourseInfo);
```

Comment:
It works correctly without typecast, same at lines 52 and 95

Reply:
fixed thanks